protractor.conf.js

```diff
@@ -153,15 +174,21 @@ exports.config = {
     }
   ],
 
+  onCleanUp(results) {
+    retry.onCleanUp(results);
+  },
+
   onPrepare() {
     if (process.env.CI) {
       const repoHash = process.env.GIT_HASH || '';
       const outputDirectory = process.env.SMART_RUNNER_DIRECTORY;
       logger.info(`SmartRunner's repoHash: "${repoHash}"`);
       logger.info(`SmartRunner's outputDirectory: "${outputDirectory}"`);
-      SmartRunner.apply({ outputDirectory, repoHash });
+      SmartRunner.apply({outputDirectory, repoHash});
     }
 
+    retry.onPrepare();
```

Comment:
please put it into the `if (process.env.CI) {}` block, we don't need it locally.